test/approve.test.js

```diff
@@ -0,0 +1,96 @@
+const { expect, assert } = require("chai");
```

Comment:
I think all test files should have their extensions simplified and be in the format `approve.js`, for example.